helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -22,40 +22,40 @@
 {{ template "chart.description" . }}
 <!-- markdownlint-enable line-length -->
 
-This page assumes general knowledge of Fiftyone Teams and how to use it.
-Please contact Voxel51 for more information regarding Fiftyone Teams.
+This page assumes general knowledge of FiftyOne Enterprise and how to use it.
+Please contact Voxel51 for more information regarding FiftyOne Enterprise.
 
 ## Important
 
 ### Version 2.0+ License File Requirement
 
-FiftyOne Teams v2.0 introduces a new requirement for a license file.
+FiftyOne Enterprise v2.0 introduces a new requirement for a license file.
 This license file should be obtained from your Customer Success Team
-before upgrading to FiftyOne Teams 2.0 or beyond.
+before upgrading to FiftyOne Enterprise 2.0 or beyond.
 
 Please refer to the
-[upgrade documentation](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/docs/upgrading.md#from-before-fiftyone-teams-version-110)
+[upgrade documentation](../docs/upgrading.md#from-fiftyone-enterprise-versions-160-to-171)
 for steps on how to add your license file.
 
 ### Version 2.2+ Delegated Operator Changes
 
-FiftyOne Teams v2.2 introduces some changes to delegated operators.
+FiftyOne Enterprise v2.2 introduces some changes to delegated operators.
 Please refer to the
-[upgrade documentation](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/docs/upgrading.md#from-fiftyone-teams-version-213)
+[upgrade documentation](../docs/upgrading.md#fiftyone-enterprise-v25-delegated-operator-changes)
```

Comment:
And this one.

Reply:
I don't think this anchor even exists - this is where we want to link (the not-outdated version of this line)

Reply:
This would be fine, but you need it to be a full URL to the github docs, because helm.fiftyone.ai doesn't have relative paths